.github/workflows/tests.yml

```diff
@@ -23,21 +20,21 @@ jobs:
         - python-version: "3.10"
           env:
             TOXENV: py
-        - python-version: pypy3
+        - python-version: pypy3.9
           env:
-            TOXENV: pypy
+            TOXENV: pypy3
```

Comment:
Do not take my word for it, but I think this `env` section may not be needed at all.

Reply:
Ok should I remove it then?

Reply:
I would remove it, and see if everything works and expected (i.e. make sure that the expected pypy version is still used by the corresponding job). If things break, you can revert, and let it be a lesson for me :)

Reply:
Yeah I have checked. the expected pypy version is running in the CI. till now all CI workflows ran just fine.

Reply:
Do you mean this section can be removed then?

Reply:
I think I originally though `TOXENV: py` was already set by default, and that it would not need to be overridden here like this to work. However, seeing as `TOXENV: py` is explicitly set on other jobs, and how `tox.ini` has a default different from `py`, I don’t think this needs changing.

Maybe it could change to `TOXENV: py` and still work, but it seems this comment is blocking the pull request, so let’s ignore this and move on.